integration-tests/src/test/java/oracle/weblogic/kubernetes/ItServerStartPolicy.java

```diff
@@ -624,36 +687,45 @@ public void testDynamicClusterReplicaCountIsMaintained() {
    * Make sure that ONLY the specified managed server is stopped. 
    * Restart the independent managed server by patching the resource definition 
    * with spec/managedServers/0/serverStartPolicy set to ALWAYS.
-   * Make sure that the specified managed server is in RUNNING state.
+   * Make sure that the specified managed server is in RUNNING state
+   * Verify that if server start policy is ALWAYS and the server is selected
+   * to start based on the replica count, it means that server is already started or is
+   * in the process of starting. In this case, script exits without making any changes.
```

Comment:
Remove the line 696 :  https://bug.oraclecorp.com/pls/bug/webbug_print.show?c_rptno=31833260. The BUG has been resolved.

Reply:
This line has not been deleted in your recent commit